.gitignore

```diff
@@ -37,6 +37,9 @@ test-results
 playwright-report
 playwright/.cache
 
+### Cypress
```

Comment:
Could we use **/downloads instead? Since we're going to need to use this pattern elsewhere